scripts/core/ui/components/dropdown-tree.tsx

```diff
@@ -47,12 +47,15 @@ export class DropdownTree<T> extends React.PureComponent<IPropsDropdownTree<T>,
         }
     }
     render() {
-        const {groups, getToggleElement} = this.props;
+        const {groups, getToggleElement, inline} = this.props;
         const onClick = () => this.setState({open: !this.state.open});
 
         return (
             <div
-                style={{display: 'flex', position: 'relative', lineHeight: 'initial'}}
+                style={inline ?
+                    {display: 'inline-flex', lineHeight: 'initial', verticalAlign: 'bottom'} :
```

Comment:
it'd be better to control the alignment from a parent element since in other cases you might need it aligned to the center for example

Reply:
This is alignment for button only, not for dropdown menu